tox.ini

```diff
@@ -1,8 +1,8 @@
 [tox]
 envlist =
-    py{27,33,34}-django17-drf{31,32},
```

Comment:
@georgemarshall It looks like python 3.2 wasn't in the environments before here or [travis.yml](https://github.com/django-json-api/django-rest-framework-json-api/pull/268/files#diff-354f30a63fb0907d4ad57269548329e3L7) Since one of your suggestions was to remove it, we can probably just leave it out?

We should probably remove any other references to python 3.2 while we are at it.